src/main/settings.py

```diff
@@ -76,16 +76,16 @@
 # Database
 # https://docs.djangoproject.com/en/2.1/ref/settings/#databases
 
-ROOT_URLCONF = "settings.urls"
+ROOT_URLCONF = "main.urls"
 
-WSGI_APPLICATION = "settings.wsgi.application"
+WSGI_APPLICATION = "main.wsgi.application"
 
 DATABASES = {
     "default": {
         "ENGINE": "django.contrib.gis.db.backends.postgis",
         "NAME": os.getenv("DATABASE_NAME", "waarnemingen_mensen"),
-        "USER": os.getenv("DATABASE_USER", "waarnemingen_mensen"),
-        "PASSWORD": os.getenv("DATABASE_PASSWORD", "insecure"),
+        "USER": os.getenv("DATABASE_USER", "postgres"),
+        "PASSWORD": os.getenv("DATABASE_PASSWORD", ""),
         "HOST": os.getenv("DATABASE_HOST", "database"),
         "CONN_MAX_AGE": 20,
         "PORT": os.getenv("DATABASE_PORT", "5432"),
```

Comment:
Als je hier de defaults aanpast naar alle waardes op "dev', en datzelfde doet in je docker-compose bij de db config, dan hoef je in de meeste situaties geen environment meer door te geven met database settings, maar dan werkt de compose-default.

Reply:
Gezien ik ook nog wel een lokaal dev hou ik het voor nu nog even zo. Maar op een later moment zet ik dat om.